data/generate_test_data.md

```diff
@@ -0,0 +1,16 @@
+## Generate keypair and container image for unit and integration tests
```

Comment:
I wonder if it is worth linking this file on the main README.md as part of a set up guide as it's easy to miss?